.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,28 @@
+name: Python package
+
+on: [push]
```

Comment:
I've not seen the list format used here before, is this not meant to be newline + tab?

Reply:
I think they are equivalent